render/src/shape_utils.rs

```diff
@@ -1222,6 +1295,31 @@ pub fn quadratic_curve_bounds(
         ))
 }
 
+pub fn cubic_curve_bounds(
+    start: swf::Point<Twips>,
+    stroke_width: Twips,
+    control_a: swf::Point<Twips>,
+    control_b: swf::Point<Twips>,
+    anchor: swf::Point<Twips>,
+) -> Rectangle<Twips> {
+    // [NA] Should we just move most of our math in this file to lyon_geom?
+    let bounds = lyon_geom::CubicBezierSegment {
+        from: lyon_geom::Point::new(start.x.to_pixels(), start.y.to_pixels()),
+        ctrl1: lyon_geom::Point::new(control_a.x.to_pixels(), control_a.y.to_pixels()),
+        ctrl2: lyon_geom::Point::new(control_b.x.to_pixels(), control_b.y.to_pixels()),
+        to: lyon_geom::Point::new(anchor.x.to_pixels(), anchor.y.to_pixels()),
+    }
+    .bounding_box();
+
+    let radius = stroke_width / 2;
+    Rectangle {
+        x_min: Twips::from_pixels(bounds.min.x) - radius,
+        x_max: Twips::from_pixels(bounds.max.x) + radius,
+        y_min: Twips::from_pixels(bounds.min.y) - radius,
+        y_max: Twips::from_pixels(bounds.max.y) + radius,
```

Comment:
it's somewhat strange that `Rectangle<Coordinate>` doesn't have a `grow` or `expand` method for this, but alas, it doesn't.

Reply:
We have so many pseudo math/geom types that all have different capabilities. I'd love to just make a ruffle-geom crate and get it over with... One day 😅